docs/index.rst

```diff
@@ -0,0 +1,15 @@
+Welcome to LIVE DATA SERVER's documentation!
+====================================
```

Comment:
I got an error due to this warning: `WARNING: Title underline too short.`

Reply:
Ouch!, how did I miss this one :question: